libs/dh/market-participant/feature-market-participant/src/components/edit.component.html

```diff
@@ -16,19 +16,21 @@
 -->
 <watt-modal
   #modal
+  autoOpen="true"
```

Comment:
nitpick: shorter syntax 


```suggestion
  autoOpen
```